R/new.R

```diff
@@ -4,11 +4,20 @@
 #'
 #' @param x A tibble-like object
 #' @param ... Passed on to [structure()]
+#' @param nrow The number of rows, guessed from the data by default
 #' @param subclass Subclasses to assign to the new object, default: none
 #' @export
-new_tibble <- function(x, ..., subclass = NULL) {
+new_tibble <- function(x, ..., nrow = NULL, subclass = NULL) {
```

Comment:
I think this needs a bit more documentation, i.e.:

* `x` must be a named list (but names are not checked for correctness).
* The `rownames` attribute is created automatically
* The `class` is set to ...